src/public/example.html

```diff
@@ -10,7 +10,9 @@
 
   <section class="container" id="visualization">
     <div id="band">
-      <audio id="buzz" src="sounds/DrainMagic.ogg"></audio>
+      <audio id="buzz">
```

Comment:
We can have the .ogg file as well as the .mp3 if we want, just have 2 <source> elements. I guess mp3 is pretty universal at this point though